front/src/reducers/osrdconf/stdcmConf/selectors.ts

```diff
@@ -0,0 +1,24 @@
+import { OsrdStdcmConfState } from 'applications/operationalStudies/consts';
+import { makeSubSelector } from 'utils/selectors';
+import buildInfraStateSelectors from 'reducers/infra/selectors';
+import buildCommonConfSelectors from 'reducers/osrdconf/osrdConfCommon/selectors';
+import { InfraStateType } from 'reducers/infra';
+import { stdcmConfSlice } from '.';
```

Comment:
Find it a bit weird with only the `'.'`
```suggestion
import { stdcmConfSlice } from './index';
```

Reply:
`from '.'` is pretty usual